lib/ice_cube/validations/monthly_by_set_pos.rb

```diff
@@ -0,0 +1,85 @@
+module IceCube
+
+  module Validations::MonthlyBySetPos
+
+    def by_set_pos(*by_set_pos)
+      return by_set_pos([by_set_pos]) if by_set_pos.is_a?(Integer)
```

Comment:
As you used the splat operator in the method's signature, I don't think it is possible for `by_set_pos` to be an `Integer`. This line thus seems unnecessary.